packages/mobile/src/screens/search-screen-v2/BpmFilter.tsx

```diff
@@ -229,14 +235,7 @@ const BpmTargetView = ({ value, setValue }: ViewProps) => {
   const onChange = useMemo(() => setValue, [])
 
   useEffect(() => {
-    if (!value && bpmTarget) {
-      setBpmTarget('')
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [value])
-
-  useEffect(() => {
-    if (minMaxValue && !isValidDiff) return
+    // if (minMaxValue && !isValidDiff) return
```

Comment:
What's this?

Reply:
nice find, so this was one of the bugs i noticed. It wont respond to updates if you select values on the "range" section, generally dont think we need this.